librosa/filters.py

```diff
@@ -494,6 +502,10 @@ def constant_q(sr, fmin=None, n_bins=84, bins_per_octave=12, tuning=0.0,
         Type of norm to use for basis function normalization.
         See librosa.util.normalize
 
+    dtype : np.dtype
+        The data type of the output basis.
+        By default, uses 64-bit complex.
```

Comment:
Potentially confusing, because np.complex64 is really two np.float32.

"By default, uses complex-valued, single-precision, floating-point format, i.e. np.complex64.
Double precision can be obtained by setting dtype=np.complex128"